debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeRecordEmitter.java

```diff
@@ -44,6 +48,9 @@
     private final PostgresConnectorConfig connectorConfig;
     private final PostgresConnection connection;
     private final TableId tableId;
+    private final boolean isJsonPlugin;
+    private Object[] cachedOldColumnValues;
```

Comment:
@jpechane What's the purpose of this "cache"? This warrants a field-level comment for sure.

Reply:
@gunnarmorling  Comment added

Reply:
@gunnarmorling Removed